.github/workflows/brew.yaml

```diff
@@ -0,0 +1,20 @@
+name: Homebrew
+
+on:
+  workflow_run:
+    workflows:
+      - "Release"
+    types:
+      - completed
+
+jobs:
+  aur:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Update Homebrew formula
+        uses: dawidd6/action-homebrew-bump-formula@v3
+        with:
+          # Required, custom GitHub access token with the 'public_repo' and 'workflow' scopes
+          token: ${{secrets.TOKEN}}
```

Comment:
And the token should probably be called `HOMEBREW_TOKEN` to make its purpose clear.